compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -80,9 +53,6 @@ void TensorDimTrackingRewriter::notifyOperationInserted(Operation *op,
 
 namespace mlir::iree_compiler::IREE::Flow {
 
-#define GEN_PASS_DEF_FORMDISPATCHREGIONSPASS
-#include "iree/compiler/Dialect/Flow/Transforms/Passes.h.inc"
-
 LogicalResult simplifyDimOps(RewriterBase &rewriter,
```

Comment:
Just for future. There is an update for `simplifyDimOps` that we need to do to truly handle dynamic cases.. we should chat about it sometime.